people/johnctronciu.py

```diff
@@ -0,0 +1,4 @@
+def printHello():
```

Comment:
I prefer camel case! Or snake case, I'm not sure 🐍

Reply:
Understandable have a great day